cicd/config/user-prefs.conf

```diff
@@ -0,0 +1,10 @@
+# These settings are to prevent popups from appearing because Splunk
+# thinks this is a fresh install.
+# https://docs.splunk.com/Documentation/Splunk/8.0.1/Admin/User-prefsconf
+[general]
+render_version_messages = 1
+checked_new_version = 8.0.1
```

Comment:
Do we need one of this for `7.2.9` as well?

Reply:
Yeah, this is something that we need to fix. Not sure how, but I have it on the list.

Reply:
But 7.2.9 works right now, so do we need this?

Reply:
I think this might need to be changed for each version to prevent popups. Not sure exactly how though...